uc-crux-llvm/src/UCCrux/LLVM/Classify.hs

```diff
@@ -123,14 +120,13 @@ classifyAssertion _sym predicate loc =
         (ExUncertain UFailedAssert)
 
 elemsFromOffset ::
-  DataLayout ->
   ModuleTypes m ->
   What4.ConcreteVal (What4.BaseBVType w) ->
   PartTypeRepr m ft ->
   Int
-elemsFromOffset dl mts offset partType =
+elemsFromOffset mts offset partType =
   let pointedTo = asFullType mts partType
-      typeSize = bytesToInteger (memTypeSize dl (toMemType pointedTo))
+      typeSize = sizeInBytes mts pointedTo 1 -- 1 = array length
```

Comment:
```suggestion
      singletonArrayLength = 1
      typeSize = sizeInBytes mts pointedTo singletonArrayLength
```

Reply:
Thanks for the comment, I've refactored the module containing this helper to have array and non-array helpers, which also clearified the code in a few other places.